lib/services/types.proto

```diff
@@ -185,6 +185,14 @@ message TLSKeyPair {
     bytes Key = 2 [ (gogoproto.jsontag) = "key,omitempty" ];
 }
 
+// JWTKeyPair is a PEM encoded keypair used for signing JWT tokens.
+message JWTKeyPair {
+    // PublicKey is a PEM encoded public key.
+    bytes PublicKey = 1 [ (gogoproto.jsontag) = "public_key,omitempty" ];
+    // PrivateKey is a PEM encoded private key. Also contains public key.
```

Comment:
does it contain public key directly?
you can derive it, but the public key isn't directly present as a second PEM blob